src/Models/ArtistsModel.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require_once('src/Entities/Artist.php');
+class ArtistsModel
+{
+    private PDO $db;
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+    public function getArtistsAlbumCount(int $artistId): Artist
+    {
+        $query = $this->db->prepare("SELECT COUNT(`albums`.`id`) AS 'album_count'
+                                                FROM `artists`
+                                                    INNER JOIN `albums`
+                                                        ON `artists`.`id` = `albums`.`artist_id`
+                                                            WHERE `artists`.`id` = :artistId;");
+        $query->setFetchMode(PDO::FETCH_CLASS, Artist::class);
```

Comment:
Using an artist entity here that only contains an album count isn't a great way to go - it makes it hard to predict what any given Artist entity will actually contain, which kinda defeats the point of using them.

In this instance, I'd suggest you either make this method return a dedicated AlbumCount entity that just contains the count and nothing else. You could also probably get away with just making it return the album count as an int